cpp/ps0007/cpp_ps0007_solution.cpp

```diff
@@ -25,9 +29,13 @@ int main()
 		cin >> dimention[0] >> dimention[1];
 	}
 	else
+	{
 		cout << "the program can't be calculated\n";
+		return 1;
+	}

```

Comment:
- [x] checked the user's input for choosing the shape
- [ ] filled in the dimensions array with positive numbers 
- [x] invoked function to calculate the area of the shape
- [x] used suitable size of dimensions arrays with the shape
- [x] used `cout::std` in a right way to print out the area